cross_corr_fft.py

```diff
@@ -165,35 +196,13 @@ def cross_corr_func(filename, if_double, version, if_plot, samp_f=fs, tar_f=freq
     actual = []
     count = 0
 
+    if(version == 0):
+        raw_data = read_data(filepath)
+        actual, ccwha, downva, count = process_data(raw_data, if_double, actual, ccwha, downva, count)
+
     for i in range(version):
```

Comment:
need an else here, version == 0 doesn't need to go into the for loop

Reply:
If version==0, then it won't enter the for loop, since for i in range(0) means that the for loop runs 0 times (so the for loop does not get executed).